mapdata/WarcraftLegacies/AbilityData/Skin/808464449.json

```diff
@@ -58,13 +53,13 @@
       "Level": 2,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Instantly teleports 72 of the player\u0027s nearby units, including the caster, to a ground unit or structure in 1250 range."
+      "Value": "After 1.5 seconds, teleports 72 of the player\u0027s nearby units, including the caster, to a ground unit or structure in 1250 range."
```

Comment:
This needs to specify that it only works on friendly units or structures, same as Archmage's Mass Teleport. I've indicated how you'd do that on the Learn tooltip but you'll need to do it for these per-level ones too.